tests/queries/0_stateless/02346_non_negative_derivative.sql

```diff
@@ -29,37 +29,37 @@ SELECT (
            FROM nnd
            LIMIT 5, 1
        );
-SELECT ts, metric, nonNegativeDerivative(metric, ts) OVER (PARTITION BY id>3 ORDER BY ts ASC Rows BETWEEN 1 PRECEDING AND 1 FOLLOWING) AS deriv FROM nnd;
+SELECT ts, metric, nonNegativeDerivative(metric, ts) OVER (PARTITION BY id>3 ORDER BY ts, metric ASC Rows BETWEEN 1 PRECEDING AND 1 FOLLOWING) AS deriv FROM nnd;
```

Comment:
Does the test become flaky if no order by `metric`?

Reply:
yes, the output order become unpredictable in that case